demo/netops-demo/values.yaml

```diff
@@ -0,0 +1,65 @@
+generate:
+  image:
+    repository: pavius/netops-demo-py
```

Comment:
If possible - move to `iguaziodocker`

Reply:
I will do this once we're done - for no reason other than possibility of littering iguaziodocker